products.php

```diff
@@ -0,0 +1,31 @@
+<?php
+require_once ('vendor/autoload.php');
+use FurnitureStoreApi\Services\HeaderService;
+HeaderService::setHeaders();
+use FurnitureStoreApi\Products\ProductHydrator;
+use FurnitureStoreApi\Services\ResponseService;
+use FurnitureStoreApi\Services\DbConnection;
+
+$db = DbConnection::setConnection();
+$query = $db->prepare('SELECT MAX(`category_id`) FROM `products`');
+$query->execute();
+$maxCategory = $query->fetch();
+$maxCategory = $maxCategory[0];
+try
+{
+    $jsonString = [];
+    if ($_GET['cat']>$maxCategory)
+    {
+        ResponseService::makeResponse('Invalid category id', $jsonString, 400);
```

Comment:
Another way of doing this is to create your own custom exception type and throw it here: I can show you how to do this.